src/lib.rs

```diff
@@ -64,6 +64,8 @@ pub struct Metadata {
     pub authors: Cow<'static, str>,
     /// The URL of the crate's website
     pub homepage: Cow<'static, str>,
+    /// The support information
+    pub supports: Cow<'static, str>,
```

Comment:
We should probably be using `Option<_>` rather than using an empty string as a sentinel

Reply:
(yes, we check for empty strings in other cases but those are meant to be populated by `metadata!` and likely we shouldn't even be doing that...)